src/components/groups/group.middlewares.ts

```diff
@@ -9,6 +9,7 @@ import { RoleType, User } from '../users/user'
 import { Group } from './group'
 import { asyncWrapper } from '../../util/asyncWrapper'
 import sendMessage from '../../util/sendMessage'
+import {sendEmail} from '../../util/sendEmail'
```

Comment:
Please add some spaces around the import like this:
`import { sendEmail } from '../../util/sendEmail'`